lib/gooddata/lcm/actions/collect_dynamic_schedule_params.rb

```diff
@@ -57,8 +57,15 @@ def call(params)
             GoodData.logger.debug("Processing row: #{safe_to_print_row}")
             results << safe_to_print_row
 
-            client_id = row[client_id_column] || :all_clients
-            schedule_name = row[schedule_title_column] || :all_schedules
+            fail "Column #{client_id_column} doesn't exist in the input." unless row.key?(client_id_column)
```

Comment:
beeing no ruby knowledgeable but I am a bit suspect whether this is the correct way of handling the situation

Reply:
In this situation, It will make sure the input should be valid before running. In this case, make sure exist 'client_id_column' in the input.